sql_scripts/filter_answer_view.sql

```diff
@@ -17,8 +17,7 @@ from (
     is_accepted,
     score,
     answer_date,
-    link,
```

Comment:
Removal of `link` is not expected. Check then run tests to make sure this appears.